roles/rabbitmq/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# set to [] to disable.
+rmq_plugins:
+  # you probably want at least one user tagged with administrator to use this plugin
+  - rabbitmq_management
```

Comment:
Still not decided/sure if we should ship the role with this plugin enabled or not (empty `[]`).
Theoretically, installing additional plugin `rabbitmq_management` by default might be not desired/expected by users behavior.